### PR TITLE
Update compose and environment vars

### PR DESCRIPTION
**Pull request by georgemjohnson11**
This with a minor update to 3Duf which allowed me to successfully build and setup using `docker-compose --env-file Fluigi-Cloud/.env up --build`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
     
+    neptune:
+      build: 
+        context: .
+      ports:
+        - 8080:8080
+        - 3000:3000
+      links:
+        - "backend_database"
+      depends_on:
```
Comment (georgemjohnson11) [inline docker-compose.yml:12]: This prevents Neptune from coming up before other services start

File: setup.sh [hunk 2]
```diff
@@ -4,15 +4,15 @@ git submodule update --recursive
 cd ./Fluigi-Cloud
 git fetch origin dev
 git checkout dev
-npm install
+npm install --legacy-peer-deps
```
Comment (georgemjohnson11) [inline setup.sh:7]: This stops conflicts from happening when node wants you to upgrade packages

File: Dockerfile [hunk 3]
```diff
@@ -64,7 +64,7 @@ RUN curl -sL --location https://deb.nodesource.com/setup_15.x | bash -
 RUN apt-get -y install nodejs
 
 # Pyenv for our baseline python environment for poetry later on.
-RUN git clone git://github.com/yyuu/pyenv.git .pyenv
+RUN git clone https://github.com/yyuu/pyenv.git .pyenv
```
Comment (georgemjohnson11) [inline Dockerfile:67]: git URLS require SSH keys which aren't needed

File: Fluigi-Cloud/.env [hunk 4]
```diff
@@ -1,8 +1,8 @@
-NEPTUNE_MONGOURL="mongodb://root:rootpassword@localhost:27017/?authMechanism=DEFAULT"
+NEPTUNE_MONGOURL="mongodb://root:rootpassword@backend_database:27017"
 NEPTUNE_SESSIONSECRET="randomsecret"
-NEPTUNE_AWS_URL="localhost:9000"
+NEPTUNE_AWS_URL="s3local:9001"
```
Comment (rkrishnasanka) [inline Fluigi-Cloud/.env]: I believe `9001` would be the web console and 9000 is the API endpoint

**Review by rkrishnasanka — CHANGES_REQUESTED** [review]
@georgemjohnson11 If the 9001 port is correct, I think we can do the merge.

**Pull request merged**